src/client.js

```diff
@@ -19,7 +22,7 @@ if (window.SIMORGH_DATA.path === window.location.pathname) {
   });
 } else {
   // eslint-disable-next-line no-console
```

Comment:
is this still needed? logger is not a console 😄

Reply:
```suggestion
```